packages/grafana-data/src/utils/moment_wrapper.ts

```diff
@@ -96,3 +97,15 @@ export const toDuration = (input?: DurationInput, unit?: DurationUnit): DateTime
 export const dateTime = (input?: DateTimeInput, formatInput?: FormatInput): DateTime => {
   return moment(input as MomentInput, formatInput) as DateTime;
 };
+
+export const dateTimeFromTimeZone = (
```

Comment:
Just a nitpicking regarding the naming. Probably `dateTimeWithTimeZone` or `dateTimeInTimeZone` would make more sense to me.

Reply:
How about `dateTimeForTimeZone`?

Reply:
Yeah that works too. Btw what will happen if you call `dateTimeFromTimeZone` without parameters?

Reply:
It will call dateTime without parameters resulting in `moment()` I believe or do you see any catch?

Reply:
No you are right, I thought maybe there was something special going on but yeah it will just be like calling `moment()`